Externals_POP.cfg

```diff
@@ -1,5 +1,5 @@
 [CVMix]
-tag = v0.93-beta
+tag = v0.97b-beta
```

Comment:
Do you want to switch to [`v0.98-beta`](https://github.com/CVMix/CVMix-src/releases/tag/v0.98-beta)? It doesn't modify anything that POP uses, but adds support for [building via `CMake`](https://github.com/CVMix/CVMix-src/pull/88). On one hand, you've done a lot of testing with the code as stands... on the other hand, this version should be bit-for-bit identical and it's nice to use the latest versions when available. I'm happy either way.

Reply:
Changed to v0.98-beta